src/routes.js

```diff
@@ -2,7 +2,7 @@ import React from 'react';
 
 import routesPaths from 'constants/routesPaths';
 import HomePage from 'pages/HomePage';
-import LoginPage from 'pages/LoginPage';
+import LoginPage from 'pages/LoginPage/LoginPage';
```

Comment:
if you are using a folder structure, is good practice to call the main component inside the folder `index.jsx`, and then you can import it as: `import LoginPage from 'pages/LoginPage';`

Reply:
Nice, thanks. Renamed every component/page so it follows this practice.